source/acquaman/AMTimedScanActionController.h

```diff
@@ -38,6 +38,7 @@ Q_DECLARE_METATYPE(AMScanActionControllerBasicFileWriter::FileWriterError)
 #define AMTIMEDSCANACTIONCONTROLLER_FILE_ALREADY_EXISTS 265003
 #define AMTIMEDSCANACTIONCONTROLLER_COULD_NOT_OPEN_FILE 265004
 #define AMTIMEDSCANACTIONCONTROLLER_UNKNOWN_FILE_ERROR 265005
+#define AMTIMEDSCANACTIONCONTROLLER_FAILED_TO_WRITE_FILE 265006
```

Comment:
@davidChevrier @dretrex 
I have a question here. We use the same error codes in `AMStepScanActionController`, `AMTimedScanActionController`, and `REIXSXESScanActionController`. But `SGMFastScanActionController` has different codes. 

I think we should use different codes. Thought?